README.md

```diff
@@ -1,2 +1,140 @@
-# primazactl
-primazactl is a simple Command Line Application for Primaza Administrator
+# Table of Contents
+ - [Introduction](#introduction)
+ - [Tool options](#tool-options)
+ - [Building the tool ](#building-the-tool)
+ - [Running the tool](#running-the-tool)
+   - [Pre-reqs](#pre-reqs)
+   - [Command options](#command-options)
+     - [Positional paramters](#positional-parameters)
+     - [Flags](#flags)
+ - [Testing](#testing) 
+ - [Other useful make targets](#other-useful-make-targets)
+
+# Introduction
+```primazactl``` is a simple Command Line Application for Primaza Administrators.
```

Comment:
If on a single line, code block may be simplified

```suggestion
`primazactl` is a simple Command Line Application for Primaza Administrators.
```

Reply:
Good to know, updated that in lots of places